utils/jwtGeneration/jwtHelper.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable camelcase*/
+const KJUR = require("jsrsasign");
+const fs = require("fs");
+const JSONWebKey = require("json-web-key");
+const jose = require("node-jose");
+const JWK = jose.JWK;
+const JWE = jose.JWE;
+const { find, keys, assign, flow, map } = require("lodash/fp");
+const yaml = require("js-yaml");
+const { refineMetadata } = require("../refineMetadata");
+
+const getKeyByUse = (json, useCase) =>
+  flow(
+    keys,
+    map(kid => assign(json[kid], { kid })),
+    find(keyObject => keyObject.use === useCase)
+  )(json);
+
+const keysYaml = yaml.safeLoad(
+  fs.readFileSync("./utils/jwtGeneration/keys.yml", "utf8")
```

Comment:
I think putting this file in the project root will make things easier when we eventually grab this file out of S3.

Reply:
Also, I think we should introduce another environment variable here too. Runner call it something like `KEYS_FILE`. We should do the same even if we just default to this for now.